modules/platform/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/rendering/operations/RenderDocumentFeed.java

```diff
@@ -66,8 +66,8 @@
     protected String charset = "UTF-8";
 
     @OperationMethod
-    public Blob run(DocumentModelList docs) throws OperationException, RenderingException, TemplateException,
-            IOException {
+    public Blob run(DocumentModelList docs)
+            throws OperationException, RenderingException, TemplateException, IOException {
```

Comment:
I'm pretty sure that Eclipse doesn't indent like that

Reply:
It does :smile:

Reply:
I'm using the Eclipse formatter with the Nuxeo Eclipse config so...

Reply:
Yes indeed, my mistake. Must be something new in it...